var/spack/repos/builtin/packages/r-bigalgebra/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RBigalgebra(RPackage):
```

Comment:
It looks like this package was removed from CRAN. Any idea why? Archive versions are still available, but no new versions.

Reply:
Don't know why the package was removed from CRAN.

Reply:
The package status can be seen at:
[CRAN Package Check Results for Package bigalgebra](https://cran-archive.r-project.org/web/checks/2020/2020-02-01_check_results_bigalgebra.html)